calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
```

Comment:
Could you use this array of operators to check that the user has submitted a valid operator?